modules/stanford_soe_helper_news/stanford_soe_helper_news.install

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @file
+ */
+
+
+/**
+ * Implements hook_install().
+ */
+
+function stanford_soe_helper_news_install() {
+
+  // Create a bean link block
+  // See https://www.drupal.org/node/1436186
+  // and http://api.drupalecommerce.org/api/misc/bean%21bean.test/function/BeanTests%3A%3AtestBeanAPI/7.x
+  
+  $text =  l(t('See more news'), '/news/recent-news',
+    array('attributes' => array('class' => array('more-link'))));
+
+  $bean = bean_create(array('type' => 'stanford_large_block'));
+  $bean->label = 'SoE News Page Link';
+  $bean->title = '';
+  $bean->delta = 'soe_news_link';
+  $bean->field_s_large_block_body = array(
+    LANGUAGE_NONE => array(
+      0 => array('value' => $text, 'format' => 'content_editor_text_format')));
+  $bean->save();
+}
+
+
+/**
+ * Implements hook_requirements().
+ */
+function stanford_soe_helper_news_requirements($phase) {
```

Comment:
Also, it would be nice to provide a link to the bean page so the user can delete it through the ui as well.